internal/captain/command.go

```diff
@@ -920,18 +885,15 @@ func childCommands(cmd *Command) string {
 }
 
 func (c *Command) LogArgs() {
-	children, err := c.FindChildren(os.Args[1:])
+	child, err := c.FindChild(os.Args[1:])
```

Comment:
https://github.com/ActiveState/cli/pull/3382#discussion_r1673003268

Reply:
For some reason all of my comments were `Pending` 🤦